pallets/runtime/src/asset.rs

```diff
@@ -1746,8 +1759,8 @@ impl<T: Trait> Module<T> {
     }
 
     /// Get the asset `id` balance of `who`.
-    pub fn balance(ticker: Ticker, did: IdentityId) -> T::Balance {
-        Self::balance_of((ticker, did))
+    pub fn balance(ticker: &Ticker, did: &IdentityId) -> T::Balance {
+        Self::balance_of(&ticker, &did).balance
```

Comment:
Do you really need second nested references here? `ticker` and `did` are already references.